letters/modernized/05-chapter-4.adoc

```diff
@@ -399,4 +399,5 @@ praise, and glory in the highest, forever and evermore, for he is worthy, worthy
 says my soul.
```

Comment:
Line 379 starts a PS, but it's before his signature, so I wasn't sure if I should make it [.postscript] or not?